src/Uno.UI/UI/Xaml/Media/Brush.skia.cs

```diff
@@ -249,15 +249,16 @@ private static IDisposable AssignAndObserveXamlCompositionBrush(XamlCompositionB
 		{
 			var disposables = new CompositeDisposable();
 
-			var compositionBrush = brush.CompositionBrush;
+			//TODO Uno: CompositionBrush property is currently unsupported
+			//var compositionBrush = brush.CompositionBrush;
```

Comment:
Should we log / raise recoverable error?

Reply:
Actually we can do fallback to `FallbackColor` - will update this.